Moonrake/Program.cs

```diff
@@ -10,6 +10,7 @@ class Program
     {
         static void Main(string[] args)
         {
+
```

Comment:
Do we really need an extra line return here?

Reply:
Yes, WE DO!!

Reply:
it workssss